### DISCUSSION TITLE
Sending and receiving email

### DISCUSSION THREAD

**Discussion by EinStyn** [emailjs-com/help#56] [Q&A]
I have a contact form I used emailJS to build. Now, when a visitor inputs their details on the form and sends it, I receive the message, and they also receive a copy when they check "sent" in their mailbox. I want only me to receive the message, and the visitor to not have a copy of the message. Do you understand?

**Comment by xr0master** [comment]
If you send an email to yourself, then your users obviously cannot receive any copies unless you send them yourself.

**Comment by EinStyn** [comment]
hmmm. It seems you did not understand what i mean. Let me paraphrase:
I created a contact form using emailJS. Currently, when a visitor submits their details via the form, I receive the message, and they also get a copy in their sent mailbox. However, I want to receive the message exclusively, and the visitor should not receive a copy. Is this clear?

**Reply by xr0master** [reply]
This is impossible. The email service doesn't work like that.
You most likely use your own email for testing, and therefore receive the email and also can see it in the sent ones.

**Reply by EinStyn** [reply]
hmmmm. Frankly as a programmer i do not believe in impossibilities. There should be a workaround yeah?
